src/commonMain/kotlin/fr/acinq/eclair/io/Peer.kt

```diff
@@ -405,6 +407,30 @@ class Peer(
                     val htlcInfos = db.channels.listHtlcInfos(actualChannelId, action.commitmentNumber).map { ChannelAction.Storage.HtlcInfo(actualChannelId, action.commitmentNumber, it.first, it.second) }
                     input.send(WrappedChannelEvent(actualChannelId, ChannelEvent.GetHtlcInfosResponse(action.revokedCommitTxId, htlcInfos)))
                 }
+                action is ChannelAction.Storage.StoreChannelClosing -> {
+                    val dbId = UUID.fromBytes(channelId.take(16).toByteArray())
+                    val payment = OutgoingPayment(
+                        id = dbId,
+                        recipientAmount = action.amount,
+                        recipient = PublicKey.Generator,
```

Comment:
The recipient should be us, shouldn't it? So `nodeParams.nodeId`?

Reply:
Yes (if actions.isSentToDefaultAddress)

Done in d105365